boot/freeldr/freeldr/arch/i386/xboxmem.c

```diff
@@ -89,12 +90,13 @@ XboxMemInit(VOID)
     AvailableMemoryMb = InstalledMemoryMb;
 }
 
-FREELDR_MEMORY_DESCRIPTOR XboxMemoryMap[2];
+FREELDR_MEMORY_DESCRIPTOR XboxMemoryMap[MAX_BIOS_DESCRIPTORS + 1];
```

Comment:
Why +1?

Reply:
Just the same as here: https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/arch/i386/pcmem.c;hb=4895f21efa522348742c28d6795c7d2768c74a8a#l43